strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/indexing/RepositoryIndexer.java

```diff
@@ -88,7 +89,8 @@ public void addArtifactToIndex(final RepositoryPath artifactPath)
             final ArtifactContextProducer artifactContextProducer = applicationContext.getBean(
                     ArtifactContextProducer.class, artifactPath);
             ArtifactContext artifactContext = artifactContextProducer.getArtifactContext(indexingContext,
-                                                                                         artifactPath.toAbsolutePath().toFile());
+                                                                                         new File(
+                                                                                                 artifactPath.toAbsolutePath().toUri().getPath()));
```

Comment:
Cosmetic: Could you please re-indent it so that it follows our convention? The first method parameter should be on the same line. If the parameters are chopped on multiple lines, each parameter should be on it's own line.